src/components/EventEdit.tsx

```diff
@@ -300,7 +300,7 @@ class EventEdit extends React.PureComponent<PropsType> {
             <Select
               name="journalUid"
               value={this.state.journalUid}
-              disabled={this.props.item !== undefined}
+              disabled={this.props.item !== undefined && !this.props.duplicate}
```

Comment:
Sorry I just threw this in because I didn't want to make a dedicated PR for it. It's unrelated to read only journals.

This change enables the selector when you are duplicating. Right now, you can't change the journal while duplicating an event.

Reply:
Ahh, it's in event edit, sorry I misread what's going on!